### PR TITLE
Prop editor styles

### PR DESCRIPTION
**Pull request by stubelshe**
Addition of Font Awesome as well

### CODE REVIEW

File: src/component-select/component-select.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default () => {
           ) : (
             <>
               <span>{selectedPage}</span>
-              <button onClick={editPageName}>&#x270E;</button>
+              <button className="edit-page-name" onClick={editPageName}><i class="fas fa-pen"></i></button>
```
Comment (mvolkmann) [inline src/component-select/component-select.js:71]: Do you like the font-awesome icons better than the Unicode characters for similar things?

File: src/component-select/component-select.scss [hunk 2]
```diff
@@ -10,30 +10,32 @@
...
-      font-size: 20px;
+      font-size: 15px;
       padding: 0;
+      &:hover {
+        color: $primary-orange;
+      }
     }
 
-    .page {
+    .page, .page input[type='text'] {
```
Comment (mvolkmann) [inline src/component-select/component-select.scss:29]: It seems like you are handling the input below on lines 36-38. Do you still need to change line 29?

File: src/component-select/component-select.scss [hunk 3]
```diff
@@ -10,30 +10,32 @@
...
       color: $gray4;
       display: inline-block;
       font-size: 20px;
       font-style: italic;
       font-weight: 100;
       line-height: 28px;
+      &.page input[type='text'] {
```
Comment (mvolkmann) [inline src/component-select/component-select.scss]: Shouldn't line 36 be just this?
input[type="text"] {

File: src/prop-editor/prop-editor.scss [hunk 4]
```diff
@@ -9,14 +9,22 @@
...
+  .reset {
+    width: calc(100% - 55px);
+  }
   .delete {
-    background-color: transparent;
-    border: none;
-    font-size: 24px;
-    margin-top: $spacing;
-    padding: 0;
+    width: 45px;
```
Comment (mvolkmann) [inline src/prop-editor/prop-editor.scss:20]: How do you feel about keeping properties within a rule in alphabetical order?
Also keeping rules in alphabetical order?
I think it helps me find what I'm looking for faster.

File: src/component-select/component-select.scss (hunk 2 above)
Comment (stubelshe) [reply src/component-select/component-select.scss:29]: This was so both .page and .page input would share all of these styles, minimizing replication. Then I broke our what was specific to the input on 36. Im open to a better way if you see one.

File: src/component-select/component-select.scss (hunk 3 above)
Comment (stubelshe) [reply src/component-select/component-select.scss]: It should, yes. My mistake.

File: src/prop-editor/prop-editor.scss (hunk 4 above)
Comment (stubelshe) [reply src/prop-editor/prop-editor.scss:20]: Thats fine, I can do that going forward.

File: src/component-select/component-select.js (hunk 1 above)
Comment (stubelshe) [reply src/component-select/component-select.js:71]: I do, yes. They give you a lot more control regarding icon options, and also styling.

**Pull request merged**